docker/connector/profiles/prod/eidas.xml

```diff
@@ -64,6 +64,203 @@
     <entry key="service3.skew.notonorafter">0</entry>
     <entry key="service3.metadata.url">https://eidasservice.eid.digst.dk/Metadata</entry>
 
+
+    <!-- Proxy-Service Island-->
+    <entry key="service4.id">IS</entry>
+    <entry key="service4.name">Iceland eIDAS Proxy Service</entry>
+    <entry key="service4.skew.notbefore">10000</entry>
+    <entry key="service4.skew.notonorafter">10000</entry>
+    <entry key="service4.metadata.url">NO-SERVICE-YET</entry>
+
+    <!-- Proxy-Service Finland-->
+    <entry key="service5.id">FI</entry>
+    <entry key="service5.name">IS eID-gateway</entry>
+    <entry key="service5.skew.notbefore">10000</entry>
+    <entry key="service5.skew.notonorafter">10000</entry>
+    <entry key="service5.metadata.url">NO-SERVICE-YET</entry>
```

Comment:
Er det en placeholder, burde heller kommenteres ut?

Reply:
orka ikkje endre id-ar på alt, får vera slik til island kjem med urlen sin. Dei er vel rett bak oss i løypa for å implementere...

Reply:
veldig teit xml-struktur... :vomiting_face: